db-zoningtaxlots/sql/area_zoningdistrict_create.sql

```diff
@@ -18,8 +18,6 @@ CREATE TABLE validzones AS (
 ); 
 CREATE INDEX validzones_geom_idx ON validzones USING GIST (geom gist_geometry_ops_2d);
 
-VACUUM ANALYZE;
-
```

Comment:
just curious why you dropped these but kept them in pluto files?

Reply:
Scratch that last point